src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemConf.scala

```diff
@@ -56,7 +60,7 @@ class FileItemConf(args: Seq[String]) extends ScallopConf(args) {
       " for the file and the ancestor folders that don't yet exist in the dataset")(mayNotExist)
   val format = opt[String](
     name = "format", short = 'f',
-    descr = "dcterms property format, the mime type of the file",
+    descr = s"dcterms property format, the mime type of the file (default$NB_SPACE=${NB_SPACE}application/octet-stream)",
```

Comment:
I was tempted to propose to add `.replaceAll("&nbsp;"," ")` to [this line](https://github.com/scallop/scallop/commit/88ae5d6d00ad39d717457438e9201d21e1e808ff#diff-c8d77f5821c15084b74aaeb7cbb822a6R22). I would expect a plain space in the output but would have to proof that with a new unit test. But first let us see if the original functionality gets restored, see [this issue](https://github.com/scallop/scallop/commit/88ae5d6d00ad39d717457438e9201d21e1e808ff#diff-c8d77f5821c15084b74aaeb7cbb822a6R22)